DOCS-5562 Win32 Event Log Integration Query Option Update (#14765)
DataDog/integrations-core: win32_event_log/README.md

<!-- xxz tab xxx -->
<!-- xxx tab "Logs" xxx -->

> You can use the `query`, as well as the `log_processing_rules` regex option, to filter event logs. If you are using the Logs Tailer, the `query` option has better filtering performance than the `log_processing_rules` filters. The Datadog Agent cannot see any events filtered out by the `query` option. When using the `log_processing_rules` filters, the Agent can see every event and must process and format the event before applying the regex filter, so it takes more processing, and is less performant.

Reviewer: > The Datadog Agent cannot see any events filtered out by the `query` option.

I'm wondering if this might be confusing.

> If you are using the Logs Tailer,

This is in the logs tab, do we need to specify this?

maybe something like

```suggestion
You can use the `query`, as well as the `log_processing_rules` regex option, to filter event logs. The `query` option has better filtering performance than the `log_processing_rules` filters. When using the `log_processing_rules` filters, the Agent must process and format each event before applying the regex filter, so it takes more processing, and is less performant.
```

> You can use the `query` to filter events or filter out the bulk of the events with an [XPATH or structured XML query][21]. There is an expression limit on the syntax of XPath and XML queries. For additional filtering, use `log_processing_rules` filters.

Reviewer: ```suggestion
You can use the `query` to filter events or filter events with an [XPATH or structured XML query][21]. The `query` option can reduce the number of events that are processed by `log_processing_rules`, improving performance. There is an expression limit on the syntax of XPath and XML queries.  For additional filtering, use `log_processing_rules` filters.
```

Reviewer 2: @clarkb7  and @estherk15 what is resulting text for the "query". Looks like the are a few versions. This is super important piece of information @clarkb7 and I would like to communicate efficiently and that implications and even reason bit more clear. That is why I am interested to read the whole section before commenting.

Datadog recommends creating and testing the query in Event Viewer's filter editor until the events shown in Event Viewer match what you want the Agent to collect.

> ![Filter Current Log][23]

Reviewer 3: I couldn't see this in your preview link, but what does the `!` do for this link

Author: This is the manual way to add images in READMEs that appear in GitHub and the Docs Site! We have to first add the images into the `/images` folder in this integration, and then it will appear in the preview link above 👍

Then, copy and paste the query into the Agent configuration. 

```yaml
  # collect Critical, Warning, and Error events
  - type: windows_event
    channel_path: Application
    source: windows.events
    service: Windows       
    query: '*[System[(Level=1 or Level=2 or Level=3)]]'
      
  - type: windows_event
    channel_path: Application
    source: windows.events
    service: Windows       
    query: |
      <QueryList>
        <Query Id="0" Path="Application">
          <Select Path="Application">*[System[(Level=1 or Level=2 or Level=3)]]</Select>
        </Query>
      </QueryList>
```

![XML Query][24]

> In addition to the `query` option, events can be further filtered with log processing rules. If the `query` option is specified, then the `filters` option is ignored. For more information, see the [example `conf.yaml` file][22].

Reviewer: the `filters` option is just for events not for logs.

```suggestion
In addition to the `query` option, events can be further filtered with log processing rules.
```
